weblate.md

```diff
@@ -1,83 +1,85 @@
 ## Step by step guide for Weblate
 
-Translators can make two actions on weblate: translate new strings or Review other contributor's translations. This guide will give all the info you need to perform both tasks. 
+Translators can perform two actions on Weblate: translate new strings or review the translations of others.
```

Comment:
Since the two trailing spaces are missing here. This string and the following one will be displayed the same way as before. Is this intentional or you wanted to display the strings on 2 different lines?